week1/plot_trips.R

```diff
@@ -18,25 +24,47 @@ load('trips.RData')
 ########################################
 
 # plot the distribution of trip times across all rides (compare a histogram vs. a density plot)
-
+ggplot(data = trips, mapping = aes(x = tripduration)) + geom_histogram() + scale_x_log10(label = comma) + scale_y_log10(label = comma)
+ggplot(data = trips, mapping = aes(x = tripduration)) + geom_density() + scale_x_log10(label = comma) + scale_y_log10(label = comma)
+install.packages('patchwork')
+library(patchwork)
```

Comment:
Two quick things about code style in R:
1. Generally R users don't like it when you put `install.packages()` in a script.
1. `library()` calls should go at the top of the file because putting them lower can cause namespace collisions in larger projects.